docs/gettingstarted.rst

```diff
@@ -51,6 +51,17 @@ Quickstart
    Each of these images covers a single Socorro component: processor, webapp,
    and crontabber.
 
+   .. NOTE::
+
+      If you're on Linux and want the host user id to match the user id in the
+      containers, then you should do::
+
+        $ SOCORRO_UID=$(id -u) SOCORRO_GID=$(id -g) make build
+
+
+      Probably best to make a script out of that if you do it often.
```

Comment:
This is the magic sauce that makes it all work for me. This is the point in the docs where we bring up `make build` the first time, so it seemed to make sense to put here. Maybe there are other good places?

Reply:
I spent some time trying to do two rules, but it's tricky because "make build" is a dependency for `.docker-build` and I don't know how to get it so that either rule can satisfy the requirement.

We could wrap the `${DC} build base` part in something that pulls the `my.env` environment. That's an interesting idea. We can resurrect the script you deleted when you did the config overhaul--it does this. That's an interesting idea and it uses `my.env` which we're pushing as the way to configure all things locally rather than another separate thing.

I'll try to do that.